pom.xml

```diff
@@ -15,6 +15,18 @@
     <version>999-SNAPSHOT</version>
     <packaging>pom</packaging>
 
+    <description>Quarkus - Kubernetes Native Java stack tailored for OpenJDK HotSpot and GraalVM</description>
+    <url>https://github.com/quarkusio/quarkus</url>
+
+    <developers>
+        <developer>
+            <id>quarkus</id>
+            <name>Quarkus Community</name>
+            <organization>Quarkus</organization>
+            <organizationUrl>http://quarkus.io</organizationUrl>
```

Comment:
Sorry for the nitpicking but this should be https.

Reply:
Fixed